src/herder/HerderSCPDriver.cpp

```diff
@@ -792,44 +780,74 @@ HerderSCPDriver::valueExternalized(uint64_t slotIndex, Value const& value)
         abort();
     }
 
-    // log information from older ledger to increase the chances that
-    // all messages made it
-    if (slotIndex > 2)
+    // externalize may trigger on older slots:
+    //  * when the current instance starts up
+    //  * when getting back in sync (a gap potentially opened)
+    // in both cases do limited processing on older slots; more importantly,
+    // deliver externalize events to LedgerManager
+    bool newSlot = true;
+    auto currentSlot = mApp.getHerder().getCurrentLedgerSeq();
+    // Allow older messages when in sync
+    if (slotIndex < currentSlot && trackingSCP())
     {
-        logQuorumInformation(slotIndex - 2);
+        newSlot = false;
```

Comment:
maybe call this `isLatestSlot` instead of `newSlot`; also just set it to its value directly instead of burying in conditions:
`isLatestSlot = (slotIndex < mApp.getHerder().getCurrentLedgerSeq());`